lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/LMFeedCore.swift

```diff
@@ -41,15 +41,16 @@ public class LMFeedCore {
         if tokens.success,
            let accessToken = tokens.data?.accessToken,
            let refreshToken = tokens.data?.refreshToken {
-            showFeed(accessToken: accessToken, refreshToken: refreshToken) { result in
+            showFeed(accessToken: accessToken, refreshToken: refreshToken, coreCallback: nil) { result in
                 completionHandler?(result)
             }
         } else {
             initiateLikeMindsFeed(apiKey: apiKey, username: username, userId: uuid, completionHandler: completionHandler)
         }
     }
     
-    public func showFeed(accessToken: String?, refreshToken: String?, completionHandler: ((Result<Void, LMFeedError>) -> Void)?) {
+    public func showFeed(accessToken: String?, refreshToken: String?, coreCallback: LMFeedCoreCallback?, completionHandler: ((Result<Void, LMFeedError>) -> Void)?) {
+        self.coreCallback = coreCallback
         self.setupFeed()
```

Comment:
Since, setupChat is independent method in Chat SDK, can we make the same flow here?

Reply:
And we can name the private functions as validateUser and initiate user, it will be much easier to read and understand that way

Reply:
Will be resolved in LM-11383 PR